Kuzzle/API/Controllers/IndexController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace KuzzleSdk.API.Controllers {
+  /// <summary>
+  /// Implements the "index" Kuzzle API controller
+  /// </summary>
+  public class IndexController: BaseController {
+    internal IndexController(IKuzzleApi api) : base(api) {}
+
+    /// <summary>
+    /// Creates a new index in Kuzzle via the persistence engine.
+    /// </summary>
+    public async Task<JObject> CreateAsync(string index) {
+      Response response = await api.QueryAsync(new JObject {
+        { "controller", "index" },
+        { "action", "create" },
+        { "index", index },
+      });
+
+      return (JObject)response.Result;
```

Comment:
The result is not important here, you could return nothing (like `DeleteAsync`) instead

Reply:
This is the case in SDK JS  https://docs-v2.kuzzle.io/sdk/js/6/controllers/index/create/#resolves :slightly_smiling_face:

Reply:
Then the js response should be modified in the next release.

Reply:
@Aschen : Fixed